src/utils/hashlink.ts

```diff
@@ -0,0 +1,118 @@
+import cbor from 'cbor'
```

Comment:
`HashLinkEncoder.ts` and `class HashLinkEncoder` to be consistent with other similar files/classes